exporter/awsxrayexporter/internal/translator/cause.go

```diff
@@ -486,6 +489,59 @@ func fillDotnetStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 	return exceptions
 }
 
+func fillGoStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	var line string
+	var label string
+	var path string
+	var lineNumber int
+
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	_, _ = r.ReadLine()
+	exception := &exceptions[0]
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		re := regexp.MustCompile(`^goroutine.*\brunning\b.*:$`)
+		match := re.Match([]byte(line))
+		if match {
+			line, _ = r.ReadLine()
+		}
```

Comment:
Should the goroutine ID be added to each label as a prefix?  It seems like this will result in flattening the stack trace with no indication that there were multiple goroutines involved.

Reply:
Not sure if that would be a good idea. Generally, label does not contain go routine id and if we do that then we would have to document that and make sure that it is visible to the users. I feel this would kinda add more overhead and confuse users since label are not usually interpreted that way.